src/Identity/samples/IdentitySample.PasskeyConformance/Program.cs

```diff
@@ -24,73 +25,91 @@
         });
     });
 
-builder.Services.AddIdentityCore<PocoUser>(options =>
-    {
-        // The origin can't be inferred from the request, since the conformance testing tool
-        // does not send the Origin header. Therefore, we need to explicitly set the allowed origins.
-        options.Passkey.AllowedOrigins = [
-            "http://localhost:7020",
-            "https://localhost:7020"
-        ];
-    })
+builder.Services.AddIdentityCore<PocoUser>()
     .AddSignInManager();
 
 builder.Services.AddSingleton<IUserStore<PocoUser>, InMemoryUserStore<PocoUser>>();
 builder.Services.AddSingleton<IUserPasskeyStore<PocoUser>, InMemoryUserStore<PocoUser>>();
 
+// In a real app, you'd rely on the SignInManager to securely store passkey state in
+// an auth cookie, but we bypass the SignInManager for this sample so that we can
+// customize the passkey options on a per-request basis. This cookie is a simple
+// way for us to persist passkey attestation and assertion state across requests.
+var passkeyStateCookie = new CookieBuilder
```

Comment:
We had discussed offline that some of these changes might make our implementation incompatible with the conformance testing tool. Thankfully, we can still make this work by bypassing the `SignInManager` and manually constructing the `DefaultPasskeyHandler` using per-request `PasskeyOptions`.